backend/src/db-utils.js

```diff
@@ -528,6 +528,24 @@ const queryPartnerUniqueUserAmount = async () => {
   );
 };
 
+// bridge query
+const queryBridgeUser = async (accountId) => {
```

Comment:
You lost me here with the naming. To query a bridge user you need to do `SELECT * FROM accounts WHERE account_id = ?`. The implementation clearly does not match it.

Reply:
change to `queryBridgeTokenHolder`